clients/include/testing_nrm2_strided_batched_ex.hpp

```diff
@@ -0,0 +1,184 @@
+/* ************************************************************************
+ * Copyright 2016-2020 Advanced Micro Devices, Inc.
+ *
+ * ************************************************************************ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <vector>
+
+#include "testing_common.hpp"
+
+using namespace std;
+
+/* ============================================================================================ */
+
+// Tolerance of 100 fails for complex,
+// TODO: something better than arbitrary tolerance.
+template <typename T>
```

Comment:
Is there a ticket made for all these TODOs?

Reply:
Not yet, on it!